FBSnapshotTestCase/FBSnapshotTestCase.m

```diff
@@ -74,6 +74,28 @@ - (void)setUsesDrawViewHierarchyInRect:(BOOL)usesDrawViewHierarchyInRect
   _snapshotController.usesDrawViewHierarchyInRect = usesDrawViewHierarchyInRect;
 }
 
+-(NSString *)testName
+{
+  return _snapshotController.testName;
+}
+
+- (void)setTestName:(NSString *)testName
+{
+  NSAssert1(_snapshotController, @"%s cannot be called before [super setUp]", __FUNCTION__);
+  _snapshotController.testName = testName;
+}
+
+-(void)setTrimTestFromImageName:(BOOL)trimTestFromImageName
+{
+  _snapshotController.trimTestFromImageName = trimTestFromImageName;
```

Comment:
I would guess that this also must be called after `[super setUp]` right?

Reply:
Yeah, right, sorry I missed that one also. Anything that configures the controller must be called only after the controller is instantiated, which is in the setUp method.